Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -76,13 +76,15 @@ import qualified Storage.Queries.SearchRequestForDriver as QSearchReqForDriver
 import qualified Storage.Queries.Vehicle as QVehicle
 import qualified Tools.Auth as Auth
 import Tools.Error
+import Storage.CachedQueries.TransporterConfig
```

Comment:
use qualified import as it will just import whatever is being used instead of importing whatever is available.